zwave_me_server/Dockerfile

```diff
@@ -6,21 +6,21 @@ ENV distro=focal
 WORKDIR /opt/z-way-server
 
 RUN apt-get update && \
-    apt-get install -y wget procps gpg libcurl4 logrotate tzdata
+    apt-get install -qqy --no-install-recommends \
+    ca-certificates curl libnss-mdns \
+    wget procps gpg iproute2 openssh-client logrotate
```

Comment:
_iproute2 openssh-client logrotate libnss-mdns_ are already in dependencies of zbw and z-way-server. please remove